### PR TITLE
17137 misc cleanup

### PR DESCRIPTION
**Pull request by severinbeauvais**
*Issue #:* /bcgov/entity#17137

_This is a bit of cleanup as I was already in the code trying to figure out how/if to merge search.vue (new request) and name-capture.vue (Edit NR). But basically these are non-functional changes._

*Description of changes:*
    - app version = 5.0.10
    - replaced Vue.prototype data with direct references
    - deleted obsolete app-config, etc
    - misc cleanup

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namerequest license (Apache 2.0).

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -173,7 +173,6 @@ export default class App extends Mixins(
   @Action setDisplayedComponent!: ActionBindingIF
   @Action toggleConfirmNrModal!: ActionBindingIF
   @Action setCurrentJsDate!: ActionBindingIF
-  @Action setIncorporateNowErrorStatus!: ActionBindingIF
```
Comment (severinbeauvais) [inline src/App.vue:176]: Already declared via mixin.

File: src/components/new-request/grey-box.vue [hunk 2]
```diff
@@ -23,7 +23,7 @@
       </template>
       <template v-else>
         <transition :name="i === 0 ? 'fade' : '' " mode="out-in">
-          <v-row :key="changesInBaseName+designationIsFixed+'key'+i"
```
Comment (severinbeauvais) [inline src/components/new-request/grey-box.vue:26]: This caused a code warning (something about adding booleans) 😛

File: src/components/new-request/search.vue [hunk 3]
```diff
@@ -262,18 +262,23 @@ import { BulletsColinLink } from '../common'
...
 import { GetFeatureFlag } from '@/plugins'
 
 /** This component is used for CREATING a Name Request. */
 @Component({
   components: { BulletsColinLink, NameInput }
 })
 export default class NewSearch extends Mixins(CommonMixin) {
+  // Refs
+  $refs!: {
+    selectBusinessTypeRef: FormType
```
Comment (severinbeauvais) [inline src/components/new-request/search.vue:279]: I added this to resolve a code warning about "form.reset() is not declared". Now that the code knows what this ref is, reset is declared.

File: src/main.ts [hunk 4]
```diff
@@ -30,20 +29,14 @@ async function startVue () {
...
   // Load Vuex config
   store.state.config = envConfig
 
-  // FUTURE: remove these globally scoped lists and import locally where needed
-  // Load global data
-  Vue.prototype.$allDesignationsList = AllDesignationsList
-  Vue.prototype.$conversionTypes = ConversionTypes
-  Vue.prototype.$designations = Designations
-  Vue.prototype.$requestActions = RequestActions
-  Vue.prototype.$usaStateCodes = UsaStateCodes
```
Comment (severinbeauvais) [inline src/main.ts:45]: ^^ the main cleanup in this PR

File: src/main.ts [hunk 5]
```diff
@@ -30,20 +29,14 @@ async function startVue () {
   const envConfig = await getConfig()
   const store = getVuexStore()
 
-  // FUTURE: remove these config assignments if possible
+  // FUTURE: remove this global assignment if possible
   // Load environment config
   Vue.prototype.$PAYMENT_PORTAL_URL = envConfig.$PAYMENT_PORTAL_URL
+
+  // FUTURE: remove this config assignment if possible
```
Comment (severinbeauvais) [inline src/main.ts:36]: Too complex to do properly/consistently right now.

**Review by JazzarKarim — APPROVED** [review]
LGTM! 👍

**Comment by pwei1018** [comment]
Temporary Url for review: https://namerequest-dev--pr-631-m5oahtsv.web.app

**Review by leodube-aot — APPROVED** [review]
LGTM!

**Pull request merged**